tests/countries/test_philippines.py

```
@@ -238,10 +238,11 @@ def test_bonifacio_day(self):
         name = "Bonifacio Day"
         self.assertHolidayName(
             name,
-            (f"{year}-11-30" for year in (*range(1988, 2008), *range(2011, 2050))),
+            (f"{year}-11-30" for year in (*range(1988, 2008), *range(2011, 2023), *range(2024, 2050))),
```

Ruff (0.8.2)

Line too long (103 > 99)

(E501)